src/SFA.DAS.Campaign.Web/ViewComponents/Sidebar/SidebarViewComponent.cs

```diff
@@ -30,6 +30,11 @@ public IViewComponentResult Invoke(SidebarType? type, SidebarHeaderType? headerT
                     case "findapprenticeshiptraining":
                         type = SidebarType.Employer;
                         break;
+                    case "hiringanapprentice":
+                    case "fundinganapprenticeship":
+                    case "upskillorretrain":
+                        type = SidebarType.EmployerWithoutNavigation;
+                        break;
```

Comment:
I'm not sure about this bit here, currently the sidebar is selected by the controller name but now were bringing in a lot of new controllers we might need a rethink